src/components/SideInformationsContainer/ExplanationCard/styles.ts

```diff
@@ -0,0 +1,58 @@
+import { Card } from "@mui/material";
+import styled from "styled-components";
+
+interface CloseCard {
+    isCardClosed: boolean;
+}
+
+
+export const ExplanationContainer = styled(Card)<CloseCard>`
+  
+    position: fixed;
+    z-index: ${({ isCardClosed }) => (isCardClosed ? "1" : "2")};;
```

Comment:
Why are you change the z-index?

Reply:
I am using it because it was the only way I found on how to make the card disappear. I got the idea by reading the definition on the <a href = "https://www.w3schools.com/cssref/pr_pos_z-index.asp"> documentation </a>. If there is a better alternative way on doing this I'd be happy to learn.